sql/S4_CREATE_SHARED_TABLE.sql

```diff
@@ -0,0 +1,7 @@
+-- uuid and duid act as unique identifier b/c docs can be shared to any user
+CREATE TABLE shared_documents
+(
+  PRIMARY KEY (uuid, duid),
+  uuid ULID   REFERENCES user_account(uuid) ON DELETE CASCADE,
+  duid KSUID  REFERENCES documents(duid) ON DELETE CASCADE
+);
```

Comment:
You can combine all (S1-S4) SQL scripts as `<version>_initial_schema.sql`, where version depends on the DB migration or versioning that we will use. Read my comment below.